docs/ml-collaborative-filtering.md

```diff
@@ -92,7 +92,7 @@ above) and "drop". Further strategies may be supported in future.
 <div class="codetabs">
 <div data-lang="scala" markdown="1">
 
-In the following example, we load ratings data from the
+In the following example, we load rating data from the
```

Comment:
@dsakuma, ratings seems fine (also given the link http://grouplens.org/datasets/movielens/)